### ISSUE TITLE
Test failure JIT/Regression/JitBlue/Runtime_72845/Runtime_72845/Runtime_72845.sh

### ISSUE THREAD

**Issue by v-wenyuxu** [dotnet/runtime#81314]
Run: [runtime-coreclr jitstress-isas-x86 20230121.1](https://dev.azure.com/dnceng-public/public/_build/results?buildId=144476&view=ms.vss-test-web.build-test-results-tab&runId=3113135&resultId=120704&paneView=debug)

Failed test:
```
coreclr linux x64 Checked jitstress_isas_1_x86_nosse @ Ubuntu.1804.Amd64.Open

- JIT/Regression/JitBlue/Runtime_72845/Runtime_72845/Runtime_72845.sh
- JIT/Regression/VS-ia64-JIT/V1.2-M01/b14324/b14324/b14324.sh

coreclr windows x64 Checked jitstress_isas_1_x86_nosse3_4 @ Windows.10.Amd64.Open

- JIT\\Regression\\VS-ia64-JIT\\V1.2-M01\\b14324\\b14324\\b14324.cmd
- JIT\\Regression\\JitBlue\\Runtime_72845\\Runtime_72845\\Runtime_72845.cmd
- JIT\\Directed\\coverage\\oldtests\\Desktop\\callipinvoke_il_r\\callipinvoke_il_r.cmd

coreclr linux x64 Checked jitstress_isas_2_x86_nosse2 @ Ubuntu.1804.Amd64.Open

- JIT/Regression/JitBlue/Runtime_72845/Runtime_72845/Runtime_72845.sh
- JIT/Directed/tailcall/tailcall/tailcall.sh
- JIT/Regression/VS-ia64-JIT/V1.2-M01/b14324/b14324/b14324.sh

coreclr windows x64 Checked jitstress_isas_2_x86_nosse41 @ Windows.10.Amd64.Open

- JIT\\Regression\\VS-ia64-JIT\\V1.2-M01\\b14324\\b14324\\b14324.cmd
- JIT\\Regression\\JitBlue\\Runtime_72845\\Runtime_72845\\Runtime_72845.cmd
- JIT\\Directed\\tailcall\\tailcall\\tailcall.cmd
- JIT\\Directed\\IL\\PInvokeTail\\TailWinApi\\TailWinApi.cmd
- JIT\\Directed\\coverage\\oldtests\\Desktop\\callipinvoke_il_r\\callipinvoke_il_r.cmd
... ...
```
**Error message:**
```
Assert failure(PID 5501 [0x0000157d], Thread: 5501 [0x157d]): Assertion failed 'topBB->bbNum <= botBB->bbNum' in 'Microsoft.FSharp.Core.PrintfImpl+PrintfEnv`3[System.__Canon,System.__Canon,System.__Canon]:WriteSkipEmpty(System.String):this' during 'Set block order' (IL size 22; hash 0xfbd06e37; Tier0-MinOpts)

File: /__w/1/s/src/coreclr/jit/optimizer.cpp Line: 4695
Image: /datadisks/disk1/work/AF480985/p/corerun
[createdump] waitpid() returned successfully (wstatus 00000000)
/datadisks/disk1/work/AF480985/w/AFBE0954/e/JIT/Regression/JitBlue/Runtime_72845/Runtime_72845/Runtime_72845.sh: line 432:  5501 Aborted                 (core dumped) $LAUNCHER $ExePath "${CLRTestExecutionArguments[@]}"

Return code:      1
Raw output file:      /datadisks/disk1/work/AF480985/w/AFBE0954/uploads/Reports/JIT.Regression/JitBlue/Runtime_72845/Runtime_72845/Runtime_72845.output.txt
Raw output:
BEGIN EXECUTION
/datadisks/disk1/work/AF480985/p/corerun -p System.Reflection.Metadata.MetadataUpdater.IsSupported=false Runtime_72845.dll ''
[createdump] Gathering state for process 5501 corerun
[createdump] Crashing thread 0000157d signal 00000006
[createdump] Writing minidump with heap to file /home/helixbot/dotnetbuild/dumps/coredump.5501.dmp
[createdump] Written 77975552 bytes (19037 pages) to core file
[createdump] Target process is alive
[createdump] Dump successfully written in 2533ms
Expected: 100
Actual: 134
END EXECUTION - FAILED
Test Harness Exitcode is : 1
To run the test:

set CORE_ROOT=/datadisks/disk1/work/AF480985/p
/datadisks/disk1/work/AF480985/w/AFBE0954/e/JIT/Regression/JitBlue/Runtime_72845/Runtime_72845/Runtime_72845.sh
Expected: True
Actual:   False


Stack trace
   at JIT_Regression._JitBlue_Runtime_72845_Runtime_72845_Runtime_72845_._JitBlue_Runtime_72845_Runtime_72845_Runtime_72845_sh()
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)
```

**Comment by EgorBo** [comment]
cc @AndyAyersMS sounds like something related to your recent PRs?

**Comment by AndyAyersMS** [comment]
Guessing this is an old test result. This was fixed by #80975.
